### PR TITLE
Add ansible PostgreSQL installation role

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

File: ansible/install-postgresql.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+- hosts: localhost
+  gather_facts: no
+  vars:
+    instance_name: ooni-measurements-db
+    disk_size: 200
+    ram_size: 4096
+    cpu_cores: 2
+  roles:
+    - { role: eclipsis, when: api_token is defined }
```
Comment (hellais) [inline ansible/install-postgresql.yml]: What role are you referencing here?

Comment (anadahz) [inline ansible/install-postgresql.yml]: This one: https://github.com/TheTorProject/ooni-sysadmin/tree/eclipsis/ansible-roles/ansible/roles/eclipsis which is not really that well tested.

Comment (hellais) [inline ansible/install-postgresql.yml]: Should probably also merge that since this doesn't make sense without it.

Comment (anadahz) [inline ansible/install-postgresql.yml]: It makes sense though eclipsis role is not a heavy dependency to run this role; one can still use to setup the postgresql role.

**Pull request merged**